src/components/forms/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback } from 'react'
+import { View, TouchableWithoutFeedback } from 'react-native'
+import { useStyles } from '../../../theme'
+import { Box } from '../../structure'
+import { BodyText } from '../../text'
+import { shameStyles } from '../../../theme/shame-styles'
+import { Icon } from '../../images-and-icons'
+
+export interface CheckboxProps {
+  /**
+   * Label to display next to the Checkbox.
+   */
+  label: string
+  /**
+   * True if selected.
+   */
+  checked: boolean
+  /**
+   * Called when selection state changes. Should propagate change to `checked` prop.
+   */
+  onChange: (checked: boolean) => void
+  /**
+   * Additional text to aid in use.
+   */
+  helpText?: string
+}
+
+const { size } = shameStyles.checkbox
+
+/**
+ * Use when the user needs to choose a single item in a list.
+ */
+export const Checkbox: React.FC<CheckboxProps> = ({ label, checked, onChange, helpText }) => {
+  const styles = useStyles(theme => ({
+    background: {
+      backgroundColor: theme.colors.fill.background.lighter,
+      borderColor: theme.colors.fill.primary.default,
+      borderWidth: theme.border.small,
+      borderRadius: theme.radius.small,
+      width: size,
+      height: size,
+      marginVertical: (theme.font.size.regular.height - size) / 2,
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    backgroundChecked: {
+      borderColor: theme.colors.fill.accent.default,
+    },
```

Comment:
Same [comment](https://github.com/hansgx/Cubes/pull/13#discussion_r445073188) as the one from #13